src/main.rs

```diff
@@ -473,7 +497,7 @@ async fn try_reconnect(
             }
             Err(e) => {
                 log::error!(
-                    "Could not connect to the database in {} attempts. Last attempt failed with \
+                    "Could not connect to the database in {:#} attempts. Last attempt failed with \
```

Comment:
I haven't been able to find the meaning of `{:#}` in the [spec](https://doc.rust-lang.org/std/fmt/index.html#sign0) (looks like incomplete syntax). What is the meaning exactly?

Also, here it's just used for printing a number. Shouldn't it be used for `reason`?

Reply:
You are right that the error should be displayed in the alternate format.

See https://doc.rust-lang.org/std/fmt/index.html#sign0 for the spec
and in particular https://docs.rs/anyhow/latest/anyhow/struct.Error.html#display-representations for how this applies to the error type.